src/api/models/customer.py

```diff
@@ -0,0 +1,149 @@
+# coding: utf-8
+"""Customer Model."""
```

Comment:
Caps and redundant encoding:
```suggestion
"""Customer model."""
```